Fix: #969 update social login and account abstraction section (#978)
Consensys/doc.linea: docs/get-started/tooling/social-login/arcana.mdx

image: /img/socialCards/arcana.jpg
---

> ## Social Login

Reviewer: We tend to prefer not to start a page with a subheading. Can this be absorbed into the `title` in the front matter? or removed?

[Arcana Auth SDK](https://arcana.network/auth) enables web3 developers to onboard app users quickly via [social login](https://docs.arcana.network/concepts/social-login). It enables the in-app, non-custodial [Arcana wallet](https://docs.arcana.network/concepts/anwallet/) for authenticated users to sign blockchain transactions. 

> To get started, see the [Auth Quickstart Guide](https://docs.arcana.network/quick-start/) and follow the steps to integrate Web3 app. Use the appropriate SDK flavor as per your [app type](https://docs.arcana.network/sdk-installation). 

Reviewer: ```suggestion
To get started, see the [Auth Quickstart Guide](https://docs.arcana.network/quick-start/) and follow the steps to integrate a web3 app. Use the appropriate SDK flavor as per your [app type](https://docs.arcana.network/sdk-installation). 
```

Author: @jlwllmr can we use Web3.0 instead? In our docs we have been using Web3 but in consensys style guide I guess web3/Web3.0 is acceptable? Lmk. Thx

Reviewer: we use `web3`

Author: ok, changing to web3. thx

> ## In-app Arcana Wallet

Reviewer: ```suggestion
## In-app Arcana wallet
```

Sentence case for subheadings — if this is a product name that should be capitalised, feel free to ignore. Just aligning with usage elsewhere and in your docs.

The [Arcana wallet](https://docs.arcana.network/concepts/anwallet/) is a built-in, in-app, non-custodial, wallet available via the [Arcana Auth SDK](https://docs.arcana.network/concepts/authsdk). It is available for [web, mobile, and gaming apps](https://docs.arcana.network/sdk-installation). 

> The Linea chain is available out of the box for Web3 app users via the in-app Arcana wallet. It is pre-configured in the blockchain network list. Developers can select it as the [default chain](https://docs.arcana.network/setup/config-dapp-with-db/) by using the [Arcana Developer Dashboard](https://dashboard.arcana.network/). 

Reviewer: ```suggestion
The Linea chain is available out of the box for web3 app users via the in-app Arcana wallet. It is preconfigured in the blockchain network list. Developers can select it as the [default chain](https://docs.arcana.network/setup/config-dapp-with-db/) by using the [Arcana Developer Dashboard](https://dashboard.arcana.network/). 
```

feel of the wallet, developers can also control the Arcana wallet UI display to sign blockchain 
transactions.

[Learn more](https://docs.arcana.network/user-guides/wallet-ui/).

> ## Enable Arcana Wallet

Reviewer: ```suggestion
## Enable Arcana wallet
```
